validator/src/main/java/com/amazon/aoc/models/PerformanceResult.java

```diff
@@ -27,6 +27,7 @@
   @NonNull private String testingAmi;
 
   @NonNull private String dataType;
+  @NonNull private String dataMode;
```

Comment:
should terraform send datamode to validator?

Reply:
It's already being sent via config.

Reply:
https://github.com/aws-observability/aws-otel-test-framework/blob/e550c5d523133f5df82dff802cd93e51606f992e/validator/src/main/java/com/amazon/aoc/models/ValidationConfig.java#L50